src/main/java/de/metas/ui/web/product/process/WEBUI_M_Product_LotNumber_Lock.java

```diff
@@ -115,39 +144,22 @@ private void quarantineHUsForLotNo(final int lotNoLockId)
 			final int bpartnerId = firstReceipt.getC_BPartner_ID();
 			final int bpLocationId = firstReceipt.getC_BPartner_Location_ID();
 
-			husToDistribute.add(HUToDistribute.builder()
-					.hu(hu)
-					.lockLotNo(lotNoLock)
-					.bpartnerId(bpartnerId)
-					.bpartnerLocationId(bpLocationId)
-					.build());
+			husToDistribute.add(HUToDistribute.builder().hu(hu)
+					.lockLotNo(lotNoLock).bpartnerId(bpartnerId)
```

Comment:
bpartnerId shall be on a different line.
Sometimes makes sense to have couple of builder methods on the same line, but they shall be somehow logically connected. In this case I could understand to have the bpatnerId and bpartnerLocationId on the same line.
Also I would suggest to move the `build()` on a new line

Reply:
same, `hu()` shall be on it's own line. It hides there and makes it hard to read.
Btw, how it was before this change it was OK.

Reply:
Done